app/pages/admin.js

```diff
@@ -132,6 +137,45 @@ function Admin() {
     </article>
   );
 
+  let manageContent = (
+    <article className="card">
+      <h2>Organizations</h2>
+      <p>
+        View all organizations in the directory here. Toggle their visibility on
+        the feed using the switch to the right.
+      </p>
+      <div style={{ width: '100%', height: '100%', overflow: 'scroll' }}>
+        {orgs ? (
+          <>
+            {Object.keys(orgs).map((key) => (
```

Comment:
If `orgs` is an array lets just map over it directly instead of using the indexes as keys.

Reply:
Tru